x/chainmanager/types/keys.go

```diff
@@ -13,4 +13,5 @@ const (
 	RouterKey = ModuleName
 )
 
-var ParamsKey = collections.NewPrefix(0) // ParamsKey is the key to store the params in the store
+var ParamsKey = collections.NewPrefix(0)             // ParamsKey is the key to store the params in the store
+var InitialChainHeightKey = collections.NewPrefix(1) // InitialChainHeightKey is the key to store the initial chain height in the store
```

Comment:
I looked into how/where this is set and/or used and and didn't find any instance of those (I mean, no `Get` or `Set`).
Since I see you're now providing the initial height via the helper, is this variable, and the corresponding field in the Keep necessary? If not, I'd remove it

Reply:
Yeah right I missed removing the leftover references.